### ISSUE TITLE
[Keyboard Navigation - Adaptive Cards 1.0] - “Column.SelectAction”are not getting activated using both enter and space keys.[Rendering] Bug Title Here

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7815]
### Target Platforms

JS

### SDK Version

1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 21H2 (OS Build: 22000.613)
Microsoft Team's version: 1.5.00.11865 (64-bit).

**User Experience:**
Keyboard-dependent users will not be able to perform the required action on the page controls present if they are not getting activated using enter, space and any other keys hence will fail to use their functionality.

**Note:** Login Credentials must not be provided within the bug.

**Repro Steps:**

> Launch Teams app.
> Navigate to the Outage Engineering channel ([https://teams.microsoft.com/l/channel/19%3akyHVRWHDV7sJaYAfDH_-pT8gW-vYBzGtq0DSTtYZyw01%40thread.tacv2/General?groupId=26a68935-51bd-454f-9dd0-b9d0054e768c&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47](https://teams.microsoft.com/l/channel/19:kyHVRWHDV7sJaYAfDH_-pT8gW-vYBzGtq0DSTtYZyw01%40thread.tacv2/General?groupId=26a68935-51bd-454f-9dd0-b9d0054e768c&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) ).
> Navigate to any adaptive card, expand the tabs Action, On-call.
> Verify the issue.

**Actual Result:**
"Actions" and "On call contacts" buttons are not getting activated i.e., Expanded using enter, space and any other keys but is accessible only by using mouse.

**Expected Result:**
"Actions" and "On call contacts" buttons should get activated using enter, space or any other keys.

**MAS Reference:**
[MAS 2.1.1 – Keyboard](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b1F29ACF9-A3E7-414F-BD8D-7E3598D877BB%7d&file=MAS%202.1.1%20%E2%80%93%20Keyboard.docx&wdLOR=c6FA267E7-81CB-46D6-A357-F39B5ABB492F&action=default&mobileredirect=true)

Reference Links:
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Screenshots

https://user-images.githubusercontent.com/96283152/188954722-f47f9893-7250-4f0f-8a6d-58e895848155.mp4

https://user-images.githubusercontent.com/96283152/188954927-9f4c30c4-42cc-4820-9c22-eb45d70aedcd.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
@jwoo-msft Can we transfer/close this issue since we no longer own iOS?

**Comment by vagpt** [comment]
As per the above comment closing this issue.
